domains/potpry.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "personal cloud study",
+
+    "owner": {
+        "username": "potpry",
+        "email": ""
```

Comment:
![image](https://github.com/is-a-dev/register/assets/76603072/4ca15e10-4eb0-4116-b5dd-fc1096e3b195)

Please add an option for social media please and thank you. Or you may choose to add an email.